### DISCUSSION TITLE
Project page can't display normally

### DISCUSSION THREAD

**Discussion by Tinhone** [community/community#84933] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

When I entering the detailed page of project, I see that only the top bar exists on the page, the area where the project chart was originally displayed has become blank, so I can't see the details of project.
当我进入项目的详细页面时，我看到页面中只有顶栏，原本显示项目图表的区域变成了空白，因此我无法查看项目的详细信息。

Personally, I think it's like a compatibility issue.
我个人认为这像是一个兼容性问题。

### Repro

1. https://github.com/orgs/microsoft/projects/866 (Any project page can trigger this problem)
2. The problem arises immediately

\--------

1. https://github.com/orgs/microsoft/projects/866 （任何项目页面均能触发问题）
2. 问题即刻出现

### Other

Browser:
浏览器：
Mozilla Firefox v88.0

Log:
控制台日志：
```
Content Security Policy: 页面设置阻止读取位于 inline 的一项资源("script-src")。 [content.js:59:206](moz-extension://fe93d048-a1f1-4986-8549-51b5ca34379f/content.js)
Content Security Policy: 页面设置阻止读取位于 inline 的一项资源("script-src")。 [content.js:59:421](moz-extension://fe93d048-a1f1-4986-8549-51b5ca34379f/content.js)
Uncaught ReferenceError: PerformanceEventTiming is not defined
    Y web-vitals.js:1
    A web-vitals.js:1
    Y web-vitals.js:1
    84817 behaviors-b162f8ca2f90.js:8
    Webpack 7
        s
        t
        <anonymous>
        O
        <anonymous>
        _
        <anonymous>
[web-vitals.js:1:5522](https://github.githubassets.com/assets/node_modules/web-vitals/dist/web-vitals.js)
    Y web-vitals.js:1
    A web-vitals.js:1
    Y web-vitals.js:1
    84817 behaviors-b162f8ca2f90.js:8
    Webpack 8
        s
        t
        <anonymous>
        O
        <anonymous>
        _
        _
        <anonymous>
Uncaught SyntaxError: private fields are not currently supported [vendors-node_modules_tanstack_query-core_build_modern_queryClient_js-node_modules_tanstack_qu-0e879d-c3f1255c8ce9.js:1:261](https://github.githubassets.com/assets/vendors-node_modules_tanstack_query-core_build_modern_queryClient_js-node_modules_tanstack_qu-0e879d-c3f1255c8ce9.js)
Uncaught SyntaxError: private fields are not currently supported [vendors-node_modules_allex_crc32_lib_crc32_esm_js-node_modules_primer_react_lib-esm_BaseStyle-86f460-5bc4fcc8d4e6.js:6:7456](https://github.githubassets.com/assets/vendors-node_modules_allex_crc32_lib_crc32_esm_js-node_modules_primer_react_lib-esm_BaseStyle-86f460-5bc4fcc8d4e6.js)
```

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by Naimulnobel** [comment]
The issue you're experiencing seems to be related to the browser's Content Security Policy (CSP) and some JavaScript features not being supported in your current browser version.

1. **Content Security Policy (CSP)**: The CSP is a security feature that helps prevent certain types of attacks, including Cross Site Scripting (XSS) and data injection attacks. The error messages you're seeing indicate that the CSP is blocking some inline scripts from running. This could be due to an extension you have installed in your browser, or it could be a setting in the browser itself.

2. **JavaScript Features Not Supported**: The error messages "Uncaught ReferenceError: PerformanceEventTiming is not defined" and "Uncaught SyntaxError: private fields are not currently supported" suggest that some JavaScript features used by the GitHub page are not supported in your current browser version. 

Here are a few things you can try to fix the issue:

1. **Update Your Browser**: Make sure you're using the latest version of your browser. Newer versions of browsers usually have better support for the latest JavaScript features.

2. **Disable Extensions**: Try disabling any browser extensions you have installed, especially ones that might interact with web content. Some extensions can interfere with the scripts on a page and cause issues like this.

3. **Check Browser Settings**: Check your browser's settings to make sure that JavaScript is enabled and that the settings aren't blocking scripts from running.

4. **Try a Different Browser**: If none of the above solutions work, you might want to try using a different browser to see if the issue persists.

If the issue continues, it might be a bug with the GitHub website itself.
